x-pack/plugin/sql/qa/server/src/main/resources/docs/docs.csv-spec

```diff
@@ -1542,6 +1562,26 @@ null           |66.91240875912409
 // end::aggPercentileRankScalars
 ;
 
+aggPercentileRankWithPercentileConfig
+// tag::aggPercentileRankWithPercentileConfig
+SELECT
+    languages,
+    PERCENTILE_RANK(salary, 65000, 'tdigest', 100.0) AS "rank_TDigest",
+    PERCENTILE_RANK(salary, 65000, 'hdr', 3) AS "rank_HDR"
+FROM emp
+GROUP BY languages;
+
+   languages   |   rank_TDigest  |     rank_HDR
+---------------+-----------------+-----------------
+null           |73.65766569962062|80.0
+1              |73.7291625157734 |73.33333333333333
+2              |88.88005607010643|89.47368421052632
+3              |79.43662623295829|76.47058823529412
+4              |85.70446389643493|83.33333333333333
+5              |100.0            |95.23809523809524
```

Comment:
Same here - wrap the results with a `ROUND` or potentially use different args to have only 3 decimals vs 14.